coalib/bears/Bear.py

```diff
@@ -84,6 +99,13 @@ class Bear(Printer, LogPrinter):
 
     LANGUAGES = ()
     REQUIREMENTS = ()
+    AUTHOR = {}
+    AUTHOR_EMAIL = {}
+    MAINTAINERS = {}
+    MAINTAINERS_EMAILS = {}
+    PLATFORMS = {'any'}
+    LICENSE = ''
```

Comment:
it is not made by me, it was there before this PR. i plan on making it a set too, but different commit :)